### DISCUSSION TITLE
Stentura fusion not connecting to Plover 4.0.0 or Plover 3.1.1

### DISCUSSION THREAD

**Discussion by Mista-Ma** [opensteno/plover#1322] [Q&A and Support]
I have the Stenograph USB plug in and stenograph driver, but all I get is the "stenograph machine is not connected " message. Switching the machine option to "Stentura "does not make a difference. Plover 3.1.1 is also not connecting.  I am running the latest version Windows 10 if that is relevant. Anyone have any idea for a fix? Any help would truly be appreciated.

**Comment by EPLHREU** [comment]
I currently use a Stentura Fusion with Plover, and have used it both with 3 and 4.
I have the keyboard protocol set to: standard 
and the serial protocol set to: stentura
But i believe both of these are irrelevant for connecting via USB.

Firstly i would check that you have everything with plover and such setup correctly, if on Linux make sure you have permissions to access the ports and files you need to connect with your user group.... etc 

What I think you have to do is something that I have to do quite often; while plover is trying to connect and is turned on, and enabled, set to Stentura USB - I need to physically unplug the USB cable from the back of the fusion and re-connect it. After doing so, plover seems to pick it up perfectly and I can get to writing. This normally happens after power-cycling the Fusion or something like that.

**Reply by Mista-Ma** [reply]
Thank you for the reply. Its good to know that the Fusion should work in theory.  Re-plugging it in did not seem to not work in either version. Plover works fine when I set the machine as my keyboard, so I am not sure if it's an installation issue. I have successfully transferred dictionaries before, so there is no issue with the cable I am using. Plover just does not want to recognize my machine. It always says it's disconnected.   So I am thinking it's some kind of driver / plug in issue. Uninstalling and reinstalling them does not fix the issue. I am using the latest versions. 

Curiously enough, my old Stentura 8000 no longer connects with Plover 3 either. Perhaps the latest version of Windows 10 is causing some weird connectivity issue.

**Reply by gdwaynewarner** [reply]
I don't know if this will help any, but I have a copy of the manual up on Cheap and Sleazy:

[http://www.cheapandsleazy.net/filez/Stentura_Fusion_User_Guide.pdf](http://www.cheapandsleazy.net/filez/Stentura_Fusion_User_Guide.pdf)

**Reply by Mista-Ma** [reply]
Thank you again, Mr. Warner. I will see what I can dig up.
